Request/ParamFetcher.php

```diff
@@ -11,15 +11,21 @@
 
```

Comment:
It's probably a good idea, the ``ParamFetcher`` begins to be really complex...
Do you want me to include #1075 in this PR ?